src/plugins/lua/phishing.lua

```diff
@@ -399,6 +425,26 @@ local function insert_url_from_string(pool, tbl, str, data)
   return false
 end
 
+local function phishing_feed_exclusion_plain_cb(string)
+  local nelts = 0
+  local new_data = {}
+  local rspamd_mempool = require "rspamd_mempool"
+  local pool = rspamd_mempool.create()
+
+  local function phishing_feed_exclusion_elt_parser(cap)
+    if insert_url_from_string(pool, new_data, cap, nil) then
+      nelts = nelts + 1
+    end
+  end
+
+  rspamd_str_split_fun(string, '\n', phishing_feed_exclusion_elt_parser)
```

Comment:
It's better to use a function from `lua_util` than this legacy global variable.

Reply:
This [lua_util.str_split(text, delimiter)](https://rspamd.com/doc/lua/lua_util.html#f59ff1) one? I would gladly get a recommendation how to use the existing phishing exclude list or an option to use multiply `[list,list2,...]` of exclusions.

Reply:
btw, not quite clear what to do with func (3rd element) in case of using str_split?

Reply:
Yes, that's my fault - it is a specific function from this module. Please ignore.